apps/dashboard/app/models/project.rb

```diff
@@ -23,6 +23,31 @@ def lookup_table
       {}
     end
 
+    def import_to_lookup(dir)
+      # fetch "id" by opening .ondemand/manifest.yml
+      manifest_path = Pathname("#{dir.to_s}/.ondemand/manifest.yml")
+      unless manifest_path.exist?
+        Rails.logger.warn("Imported directory is not a Open OnDemand project")
+        return false
```

Comment:
Can we add some sort of class method helper that's like `self.from_directory`. I feel like that would clean some of this up. Let it return nil if it can't make a project out of the directory, and a real Project object if it can.